src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -12,31 +12,40 @@
 
 (defn account-options
   []
-  [:<>
-   [quo/drawer-top temp/account-data]
-   [quo/action-drawer
-    [[{:icon                :i/edit
-       :accessibility-label :edit
-       :label               (i18n/label :t/edit-account)
-       :on-press            #(rf/dispatch [:navigate-to :wallet-edit-account])}
-      {:icon                :i/copy
-       :accessibility-label :copy-address
-       :label               (i18n/label :t/copy-address)}
-      {:icon                :i/share
-       :accessibility-label :share-account
-       :label               (i18n/label :t/share-account)}
-      {:icon                :i/delete
-       :accessibility-label :remove-account
-       :label               (i18n/label :t/remove-account)
-       :danger?             true}]]]
-   [quo/divider-line {:container-style {:margin-top 8}}]
-   [quo/section-label
-    {:section         (i18n/label :t/select-another-account)
-     :container-style style/drawer-section-label}]
-   [rn/flat-list
-    {:data      temp/other-accounts
-     :render-fn (fn [account] [quo/account-item {:account-props account}])
-     :style     {:margin-horizontal 8}}]])
+  (let [{:keys [name customization-color emoji address]} (rf/sub [:wallet/current-viewing-account])]
+    [:<>
+     [quo/drawer-top
+      {:title                name
+       :type                 :account
+       :networks             [{:name :ethereum :short-name "eth"}
```

Comment:
there's a sub to get these network details. :wallet/network-details - it's already in this shape too. Let's use that. It will be all networks but there is some future work to do the filtering based upon network configuration 👍 For now it's best to remove mock data where possible unless we really need it

Reply:
These are existing codes from the temp data (`account-data`). I have updated the values except for the networks. I have plans to do a clean-up across the wallet ns separately as this PR scope is a bit heavy at the moment (Don't want to introduce any bugs). 😄

Reply:
Definitely, sounds good to me!